python/utils.py

```diff
@@ -187,19 +188,43 @@ def sweep_ref(I, Rmin, dR, nrow, ncol, dxy, inc, Dx=0., Dy=0., dtype_image='floa
     """
     Compute the intensity map (i.e. the image) given the radial profile I(R)=ints.
     We assume an axisymmetric profile.
+    The output image is assumed with origin in the upper left corner.
 
     Parameters
     ----------
     I: 1D float array
         Intensity radial profile I(R).
-    gridrad: array
-        Radial grid
-    inc: float
-        Inclination, degree
+    Rmin : float
+        Inner edge of the radial grid, i.e. the radius where the brightness is intensity[0].
+        **units**: rad
+    dR : float
+        Size of the cell of the radial grid, assumed linear.
+        **units**: rad
+    nrow : int
+        Number of rows of the output image.
+        **units**: pixel
+    ncol : int
+        Number of columns of the output image.
+        **units**: pixel
+    dxy : float
+        Size of the image cell, assumed equal and uniform in both x and y direction.
+        **units**: rad
+    inc : float
+        Inclination along North-South axis.
+        **units**: rad
+    Dx : optional, float
+        Right Ascension offset (positive towards East, left).
+        **units**: rad
+    Dy : optional, float
+        Declination offset (positive towards North, top).
+        **units**: rad
+    dtype : optional, str
```

Comment:
`dtype` -> `dtype_image`. Please what kind of string you expect; i.e., a numpy `dtype` specification

Reply:
Agree. Fixed